test/Microsoft.Azure.WebJobs.Host.UnitTests/Executors/FunctionExecutorTests.cs

```diff
@@ -393,6 +432,44 @@ public async Task ExecuteLoggingAsync_WithDrainModeManagerEnabled_RequestsCancel
             }
         }
 
+        [Theory]
+        [InlineData(false)]
+        [InlineData(true)]
+        public async Task ExecuteWithRetries_Test(bool invocationThrows)
+        {
+            var functionInstanceEx = (IFunctionInstanceEx)CreateFunctionInstance(Guid.NewGuid(), invocationThrows);
+            var logger = new TestLogger("Tests.FunctionExecutor");
+            var functionExecutor = GetTestFunctionExecutor();
+            // Arrange
+            HostStartedMessage testMessage = new HostStartedMessage();
+            functionExecutor.HostOutputMessage = testMessage;
+
+            int maxRetryCount = 5;
+            TimeSpan delay = TimeSpan.FromSeconds(1);
```

Comment:
This will make the test run 5 seconds? Might just make this 100 ms or something, since the actual delay doesn't matter.

Reply:
done!